cognite/powerops/client/shop_results_compare.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import logging
+from datetime import datetime
+from typing import TYPE_CHECKING, Optional, Sequence
+
+import matplotlib.pyplot as plt
+
+from cognite.powerops.client.shop_result_files import ShopYamlFile
+from cognite.powerops.utils.plotting import ax_plot_time_time_series
+
+if TYPE_CHECKING:
+    from cognite.powerops import PowerOpsClient
+
+logger = logging.getLogger(__name__)
+
+
+class ShopResultsCompareAPI:
+    def __init__(self, po_client: PowerOpsClient):
+        self._po_client = po_client
+
+    def time_series_plots(
```

Comment:
Would plot_time_series be a better name? since it is actually plotting the time series and not returning a list of time series plots I think it could be better.

Reply:
Agreed